packages/apps/hr/src/components/molecules/ActiveTimerStopwatch/ActiveTimerStopwatch.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2024 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, {useEffect} from 'react';
+import {useTranslator, Stopwatch, StopwatchType} from '@axelor/aos-mobile-core';
+import {StyleSheet} from 'react-native';
+
+const DEFAULT_TIME = 0;
+const DEFAULT_STATUS = StopwatchType.status.Ready;
+
+interface ActiveTimerStopwatchProps {
```

Comment:
Please remove all active prefix you put on components/screen that are used by both active timer and standard timers as saw together.